offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonView.groovy

```diff
@@ -83,8 +86,21 @@ class CreatePersonView extends FormLayout {
         emailField.setPlaceholder("Email address")
         emailField.setRequiredIndicatorVisible(true)
 
+        HorizontalLayout organisationLayout = new HorizontalLayout()
+        organisationLayout.setMargin(false)
+        organisationLayout.setSpacing(false)
+        organisationLayout.setSizeFull()
+
         this.organisationComboBox = generateOrganisationSelector(createPersonViewModel.availableOrganisations)
         organisationComboBox.setRequiredIndicatorVisible(true)
+        organisationComboBox.setSizeFull()
+        this.createAffiliationButton = new Button(VaadinIcons.PLUS)
+        createAffiliationButton.setHeight(organisationComboBox.getHeight(), organisationComboBox.getHeightUnits())
+        createAffiliationButton.addStyleName(ValoTheme.BUTTON_FRIENDLY)
+
+        organisationLayout.addComponentsAndExpand(organisationComboBox)
+        organisationLayout.addComponent(createAffiliationButton)
+        organisationLayout.setComponentAlignment(createAffiliationButton, Alignment.BOTTOM_LEFT)
```

Comment:
The alignment is to the bottom. I did set the size the same as for the combo box. This component does not seem to provide the actual size but the size without padding. We could wrap the button in a layout or something but I think that would make it more difficult in the resulting webpage.
I did not add spacing because I want it to be `part` of the first combo box. This way it is clear that this button belongs to the list of organisations. This decision was made because it was wished in the past that an entry would appear in the drop-down of the organisations. Since this is hard I opted for the next best thing.

Adding spacing would decouple them I would keep that as is until we have customer feedback indicating they want to have spacing there.

Reply:
the middle alignment does not work. We have a form layout and as such the caption is placed above the Combobox making the actual selection appear on the bottom.